src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientData.java

```diff
@@ -41,4 +42,7 @@
     @NotNull
     Optional<Mqtt5ServerConnectionData> getServerConnectionData();
 
+    @NotNull
+    Optional<MqttClientSslData> getSslData();
```

Comment:
Optional<MqttSslConfig> getSslConfig() should be part of the MqttClientData interface